Resources/SharedTools/SerialPorts.resource

```diff
@@ -65,32 +65,32 @@ N Number of Filtered Lines of Log File Should Contain a String
 
 String Found in Serial Log File
     [Documentation]  Check if an already saved log file from the serial port ${log_name} has a string ${string_to_find}
-    [Arguments]  ${string_to_find}  ${log_name}
+    [Arguments]  ${string_to_find}  ${log_name}  ${string_or_regex}=string
     ${path}  Get Log File Path  ${log_name}
-    ${result}  File Contains String  ${string_to_find}  ${path}
-    Should Be True  ${result}
+    ${function_result}  File Contains String  ${string_to_find}  ${path}  ${string_or_regex}
+    Should Be True  ${function_result}
 
```

Comment:
It was already a efficient keyword, thanks to that update, it became even more efficient.